packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -37,6 +36,7 @@ const restoreStateIfNeeded = EventInternals[4];
 // const flushPassiveEffects = EventInternals[5];
 // TODO: This is related to `act`, not events. Move to separate key?
 // const IsThisRendererActing = EventInternals[6];
+const act = EventInternals[7];
```

Comment:
The intention of these being "Event internals" is that they would eventually go away and be replaced by other event handling (e.g. native). Then they would all go away at once.

It probably makes more sense to expose this on `__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED` directly since this wouldn't go away as part of the same refactoring.

Unless we just expose it in the DEV build's entry point as a public API.

Reply:
I think my favorite option is to export it from the isomorphic package

Reply:
I’ll move to secret internals for now